src/main/kotlin/calculator/domain/InputTokenizer.kt

```diff
@@ -0,0 +1,40 @@
+package calculator.domain
+
+class InputTokenizer(inputString: String?) {
+    private lateinit var delimiter: String
+    private lateinit var tokens: List<String>
+
+
+    init {
+        if (inputString.isNullOrBlank()) {
+            initDefaultDelimiter()
```

Comment:
```suggestion
class InputTokenizer(inputString: String) {
    private lateinit var delimiter: String
    private lateinit var tokens: List<String>


    init {
        if (inputString.isNullOrBlank()) {
            initDefaultDelimiter()
```

inputString에 대해서는 null이 아님을 표현하는 게 좋지 않을까 하네요~ 그러면, init 메서드가 좀 더 단순해 질거에요.

> > 빈 문자열 또는 null을 입력할 경우 0을 반환해야 한다. 

에 대해 널을 허용하셧다면, InputTokenizer의 상위 객체에서 null 체크를 하여 0으로 내려주는 게 좋지 않을까 합니다.